deepfence_server/handler/scan_reports.go

```diff
@@ -1454,6 +1454,9 @@ func listScanResultsHandlerWithSeverityCounts[T any](w http.ResponseWriter, r *h
 		return nil, model.ScanResultsCommon{}, severityCounts, err
 	}
 
+	// order filter doesnot have meaning when getting sev counts
+	req.FieldsFilter.OrderFilter.OrderFields = []reporters.OrderSpec{}
```

Comment:
Why is order filter set in the first place?

Reply:
It's one api endpoint `/scan/results/vulnerability` which returns `results` and `severity counts`. Order filter required for results but not for `severity counts`